src/widgets/restFullClient/ui/OneHeaderElement.tsx

```diff
@@ -0,0 +1,114 @@
+import * as yup from 'yup';
+import { yupResolver } from '@hookform/resolvers/yup';
+
+import { useForm } from 'react-hook-form';
+import { Box, Button, TextField } from '@mui/material';
```

Comment:
```suggestion
import Box from '@mui/material/Box';
import Button from '@mui/material/Button';
import TextField from '@mui/material/TextField';
```

Это на свое усмотрение. Такой импорт просто уменьшает финальный бандл. Точно такая же история с импортом lodash

Reply:
Предлагаю вместо этого поставить `babel`